base/mqueue/mqueue.go

```diff
@@ -65,7 +66,22 @@ func WriterFromEnv(topic string) Writer {
 	return writer
 }
 
-type MessageHandler func(message kafka.Message)
+type MessageHandler func(message kafka.Message) error
+
+func MakeRetryingHandler(handler MessageHandler) MessageHandler {
+	return func(message kafka.Message) error {
+		var err error
+		b, cancel := policy.Start(context.Background())
```

Comment:
I didn't know what b is and needed to find it in code. I would rather to name it backoff.